erpnext/hr/doctype/driver/driver.js

```diff
@@ -2,4 +2,13 @@
 // For license information, please see license.txt
 
 frappe.ui.form.on('Driver', {
+	setup: function(frm) {
+		frm.set_query('transporter', function(){
+			return {
+				filters: [
+					['Supplier', 'supplier_group', 'in', ['Transporter', 'transporter']]
```

Comment:
Don't use hard-coded value "Transporter" in code. The user will never know until they read code / docs.
Let's allow all suppliers.

Reply:
@nabinhait I've made the discussed changes.